ui2/src/pages/network/Clearing.tsx

```diff
@@ -169,58 +203,84 @@ export const ClearingServiceTable: React.FC<Props> = ({ services }) => {
-                      variant="contained"
-                      onClick={() => rejectOffer(c)}
-                    >
-                      Reject
-                    </Button>
-                  </>
-                ) : (
-                  <>
-                    <Button
-                      size="small"
-                      className="ghost"
-                      variant="contained"
-                      onClick={() => withdrawOffer(c)}
-                    >
-                      Withdraw
-                    </Button>
-                  </>
-                )}
-              </Button.Group>,
-            ],
-          };
-        })}
+        rows={[
+          ...offers.map((c, i) => {
+            return {
+              elements: [
+                getTemplateId(c.templateId),
+                getName(c.payload.customer),
+                <Button.Group>
+                  {c.payload.customer === party ? (
+                    <>
+                      <ModalFormErrorHandled onSubmit={() => acceptOffer(c)} title="Accept Offer">
+                        <Form.Select
+                          label="Clearing Account"
+                          placeholder="Select..."
+                          required
+                          min={1}
+                          options={accountNames}
+                          value={clearingAccountName}
+                          onChange={(_, change) => setClearingAccountName(change.value as string)}
+                        />
+                        <Form.Select
+                          label="Margin Account"
+                          placeholder="Select..."
+                          required
+                          options={allocationAccountNames}
+                          value={marginAccountName}
+                          onChange={(_, change) => setMarginAccountName(change.value as string)}
+                        />
+                      </ModalFormErrorHandled>
+                      <Button
+                        size="small"
+                        className="ghost"
+                        variant="contained"
+                        onClick={() => rejectOffer(c)}
+                      >
+                        Reject
+                      </Button>
+                    </>
+                  ) : (
+                    <>
+                      <Button
+                        size="small"
+                        className="ghost"
+                        variant="contained"
+                        onClick={() => withdrawOffer(c)}
+                      >
+                        Withdraw
+                      </Button>
+                    </>
+                  )}
+                </Button.Group>,
+              ],
+            };
+          }),
+          ...marketOffers.map((c, i) => {
+            return {
+              elements: [
+                getTemplateId(c.templateId),
+                getName(c.payload.customer),
+                <Button.Group>
```

Comment:
do we need a button group?

Reply:
(personally I like the semantic-ui buttons better, which are just 
```
 <Button
          className="ghost"
          onClick={() => acceptOffer(c)}
/>
```

Reply:
These are now semantic buttons but some of the props may be left over from the material versions. I guess it doesn't need the `Button.Group`, I just had that since some of the other tables on this page have multiple buttons, I'll try without!